cms/models.py

```diff
@@ -244,6 +254,25 @@ class Meta:
         help_text="Image that will be used where the course is featured or linked.",
     )
 
+    def _get_player_config(self):
+        """ Get configuration for video player """
+
+        if self.video_url:
+            config = {
+                "techOrder" : ["html5"],
+                "sources" : [{
+                    "src": self.video_url
+                }]
+            }
+            try:
+                embed = get_embed(self.video_url)
+                provider_name = embed.provider_name.lower()
+                config["techOrder"] = [provider_name, *config["techOrder"]]
+                config["sources"][0]["type"] = f"video/{provider_name}"
+                return dumps(config)
+            except EmbedException:
+                return dumps(config)
```

Comment:
We can avoid the repetition here
```suggestion
            except EmbedException:
                pass
            return dumps(config)
```

Reply:
I have removed the duplicate code and rather then passing the code i have a logged the url as It could come in handy later on.The log level I used is info.